mirascope/core/base/_utils.py

```diff
@@ -115,7 +125,7 @@ def parse_prompt_messages(
 
 
 def convert_function_to_base_tool(
-    fn: Callable, base: type[BaseToolT]
+    fn: Callable, base: type[BaseToolT], __doc__: str | None = None, namespace: str | None = None
```

Comment:
we should also update this to `__namespace__` now

Reply:
OK, I just updated it.